### DISCUSSION TITLE
user auth is not working in desktop only in one device

### DISCUSSION THREAD

**Discussion by RigiBenaka** [supabase/supabase#28409] [Questions]
User auth is not working for one user in desktop but working in mobile. When checked auth logs got these error message
"DEPRECATION NOTICE: GOTRUE_JWT_ADMIN_GROUP_NAME not supported by Supabase's GoTrue, will be removed soon" and "prometheus server (0.0.0.0:9122) shut down" . Whatis this error and how to resolve?

**Comment by GaryAustin1** [comment]
Those are not likely related to your issue.
![image](https://github.com/user-attachments/assets/acae88e3-04af-4510-ac7f-530c88006b22)
I've also seen the shutdown message in the past.  I believe prometheus server is related to logs.

I would look for other causes of your issue.

**Comment by NightLock305** [comment]
Did you ever figure this out?
